src/controllers/restful/PaymentController.ts

```diff
@@ -174,3 +204,13 @@ export async function notifyPaymentStatusToAPINotifica(): Promise<boolean> {
   // TODO: [#158176380] Gestione della conferma di attivazione di un pagamento
   return false;
 }
+
+// Generate a Session Token to follow a stream of requests
+function generateCodiceContestoPagamento(): Either<
+  Error,
+  CodiceContestoPagamento
+> {
+  return CodiceContestoPagamento.decode(uuid.v1()).mapLeft(() => {
+    return Error();
```

Comment:
why empty the error ? can't we just `return CodiceContestoPagamento.decode(uuid.v1())`: ?

Reply:
decode provide a ValidationError[] on left that is not compatible with ReadonlyArray[]. For this reason I provide an Error rather than decode directly

Reply:
ok but in this way we lose the validation error message. at least we should convert it to a string using message = readableReporter(errs) then return Error(message)